### ISSUE TITLE
nuxt-vite-legacy returns unexpectedly

### ISSUE THREAD

**Issue by kingyue737** [IlyaSemenov/nuxt-vite-legacy#5]
The console displays:
```
 WARN  nuxt-vite-legacy didn't find legacy-polyfills-legacy chunk. Legacy build will not work. 
```

But I only use `vite-plugin-legacy` to generate modern polyfills in my project with `renderLegacyChunks: false` in config. The plugin should still mutate the reference in the manifest instead of aborting.

**Comment by IlyaSemenov** [comment]
Can you please elaborate? Which reference should be mutated and how?

**Comment by kingyue737** [comment]
Firstly I don't understand why `renderLegacyChunks` is omited. https://github.com/IlyaSemenov/nuxt-vite-legacy/blob/b982c7f654af62ee0a923b571a9bb1f5bc3d2bec/src/module.ts#L4

Some users (like me) generate polyfills via `vite-plugin-legacy` for modern but old browsers (like chrome 69).
The config is generally like the following one:
```ts
legacy({
  modernPolyfills: true,
  renderLegacyChunks: false,
}),
```

With the above config, manifest key of polyfills is `vite/legacy-polyfills`, instead of `vite/legacy-polyfills-legacy` in source code:
https://github.com/IlyaSemenov/nuxt-vite-legacy/blob/b982c7f654af62ee0a923b571a9bb1f5bc3d2bec/src/module.ts#L52

```ts
'vite/legacy-polyfills':
   { resourceType: 'script',
     module: true,
     prefetch: true,
     preload: true,
     file: 'polyfills.b7972713.js',
     isEntry: true,
     src: 'vite/legacy-polyfills' } }
```

**Comment by IlyaSemenov** [comment]
Fair point, this use case didn't occur to me. This was addressed in version 1.1.0.
